capstone/src/main/java/com/google/sps/data/Room.java

```diff
@@ -48,29 +66,71 @@ public void setTitle(String title){
         this.entity.setProperty(Room.TITLE_PROPERTY_KEY, title);
     }
 
+    public String getDescription() {
+        return (String) this.entity.getProperty(Room.DESCRIPTION_PROPERTY_KEY);
+    }
+
+    public void setDescription(String description) {
+        this.entity.setProperty(Room.DESCRIPTION_PROPERTY_KEY, description);
+    }
+
     @SuppressWarnings("unchecked")
-    public List<Key> getAllFollowers() {
+    public List<Key> getAllFollowers() { // For Andrew: Your data model showed these as actual User objects. Was that on purpose?
```

Comment:
Communication should happen through channels other than checked in code. This comment should be removed (same for all other instances)